_layouts/events.html

```diff
@@ -39,7 +37,7 @@ <h3 class="alt text-gray">{{event.title}}</h3>
         {{event.venue}}
        </div>
        <div class="speaker">
-        {{event.speaker}}
+        {{event.speaker}}  is speaking
```

Comment:
To be clear, I'd like to see this either left the way it was or properly switch between 'is' and 'are' depending on the number of speakers.